src/lamplib/tasks/run_tests.py

```diff
@@ -110,7 +110,7 @@ def resmoke_test(env, suites, mongo_dir, is_cnats):
         # Default download location for MongoDB binaries.
         env["PATH"] += ":" + os.path.join(mongo_dir, "bin") + ":" + mongo_dir
 
-    evg_venv_dir = os.path.join(workdir, "build", "venv")
+    evg_venv_dir = os.path.join(os.environ['SCRIPTS_DIR'], "venv")
```

Comment:
Thanks to the -g flag, there is no guarantee that this venv exists which may affect people not using the automatically downloaded genny toolchain on weird platforms. Should we do anything about that?

If we're going to handwave this issue, then let's add a clean error message in the cases where the env variable is empty string/undefined. Also, can we rename it to LAMP_VENV_DIR since we're exporting it. You can leave all the other genny scripts alone.